src/System.Private.CoreLib/shared/System/StringComparer.cs

```diff
@@ -61,7 +61,7 @@ public static StringComparer Create(CultureInfo culture, CompareOptions options)
         {
             if (culture == null)
             {
-                throw new ArgumentException(nameof(culture));
+                throw new ArgumentNullException(nameof(culture));
```

Comment:
A test for this is conspicuously missing from https://github.com/dotnet/corefx/blob/8455fd2d9bd46fc8b97fe86f907d2a8ae748443f/src/System.Runtime/tests/System/StringComparerTests.cs#L13-L16.  Would you mind opening a PR in corefx coverage for this?  Thanks.

Reply:
Created as https://github.com/dotnet/corefx/pull/40902